Foundation/FileManager.swift

```diff
@@ -618,7 +793,27 @@ open class FileManager : NSObject {
         })
         return contents
     }
-    
+
+#if os(Windows)
+    private func join(_ prefix: String, _ suffix: String) -> String {
```

Comment:
Absolutely!  Any suggestions for the name?

Reply:
Perhaps `private func joinPath(prefix: String, suffix: String) -> String`?
I think there is value having labels on the prefix and suffix so that just in case future maintenance does not confuse the order or meaning of the function